mne/preprocessing/_csd.py

```diff
@@ -26,7 +26,7 @@
 from ..channels.interpolation import _calc_g, _calc_h
 
 
-def _prepare_G(G, lambda2):
+def _prepare_G(G, lambda2):  # noqa: N802
```

Comment:
why is this one excepted (and also `_check_Xy`) but `_fit_q` and `_do_tsss` are not?  I don't get the logic for which ones you're changing and which ones get `noqa` exceptions.

Reply:
> I don't get the logic for which ones you're changing and which ones get noqa exceptions.

For the most part I tried to change them, unless there was a good reason not to. Here it's because I've assumed that it's called `G` in the papers, but that's probably true of the dipole stuff, too. So I'll change this one.

The `_check_Xy` ones I did not change because maintaining the sklearn convention (where I assume this was taken from) seemed worth using as an exception.